plugin/src/main/java/at/helpch/chatchat/config/holders/SettingsHolder.java

```diff
@@ -16,6 +16,9 @@
     private PMFormat recipientFormat = DefaultConfigObjects.createPrivateMessageRecipientFormat();
     private PMFormat socialSpyFormat = DefaultConfigObjects.createPrivateMessageSocialSpyFormat();
 
+    private String itemFormat = DefaultConfigObjects.createItemFormat();
```

Comment:
is there much point having a default config objects method for a literal?

Reply:
Not really no. Lmao. I just followed you guys. But I guess having just a default value would've made more sense. will change that.